adapters/ix/ix.go

```diff
@@ -17,14 +19,17 @@ import (
 	"github.com/prebid/prebid-server/errortypes"
 )
 
+// maximum number of bid requests
+const requestLimit = 20
+
 type IxAdapter struct {
 	http *adapters.HTTPAdapter
 	URI  string
 }
 
-// used for cookies and such
+// Name is used for cookies and such
 func (a *IxAdapter) Name() string {
-	return "ix"
+	return fmt.Sprintf("%s", openrtb_ext.BidderIx)
```

Comment:
You could also do this with a simple `return string(openrtb_ext.BidderIx)`

Reply:
changed in e56b0b0